### DISCUSSION TITLE
How can I either change the `hostname` variable within the module _or_ run arbitrary commands within a `format` string?

### DISCUSSION THREAD

**Discussion by wesbaker** [starship/starship#4068] [Q&A] [answered]
I'd like to replace the output of the `hostname` module with the response from `hostname -A`. I'd be happy to either modify the `hostname` variable or the `prompt`, but I'm not sure how to go about it.

**Comment by chipbuster** [comment, marked as answer]
The hostname module doesn't use the hostname command. If you'd like to run a custom command, you can use [a custom module](https://starship.rs/config/#custom-commands).

By default, it'll display at the end, so you may want to modify the top-level `format` string to place the module where you want.
